### PR TITLE
feat(tokens): add new tokens, migrate overlay token

### PR DESCRIPTION
**Pull request by saurabhdaware**
This PR was opened by the Token Upload GitHub action. It updates source token files based on the payload from Figma Plugin.

### Summary by CodeRabbit

- Style: Updated the `popup` background color in the banking and payment themes to improve visual appearance.
- Style: Updated the `overlay` background color in the backdrop components to enhance visual consistency.
- Refactor: Restructured the `Colors` type by removing the `overlay` property and moving it under the `action` property.
- Documentation: Provided a warning message and code snippet for replacing the `theme.colors.overlay.background` token with `theme.colors.surface.overlay.background[800]`.
- Chore: Recommended searching for usages of `colors.overlay.background` in the repository to confirm if it's being used.

### CODE REVIEW

**Comment by saurabhdaware** [comment]
The checks will pass once we merge #1772 into this

**Review by kamleshchandnani — APPROVED** [review]

File: packages/blade/src/tokens/theme/theme.ts [hunk 1]
```diff
@@ -146,7 +146,6 @@
     overlay: Record<'background', Record<400 | 800, string>>;
     popup: Record<'background', string>;
   };
-  overlay: Record<'background', string>;
   action: {
     background: Omit<ActionVariants, 'link'>;
     border: Omit<ActionVariants, 'link'>;
```
[the remark this answers is not in the document]
Comment (saurabhdaware) [reply packages/blade/src/tokens/theme/theme.ts:151]: there is semicolon already

File: packages/blade/src/tokens/theme/theme.ts [hunk 2]
```diff
@@ -146,7 +146,6 @@ export type Colors = {
     overlay: Record<'background', Record<400 | 800, string>>;
     popup: Record<'background', string>;
   };
```
[the remark this answers is not in the document]
Comment (saurabhdaware) [reply packages/blade/src/tokens/theme/theme.ts:148]: it is expected

**Review by chaitanyadeorukhkar — APPROVED** [review]

File: packages/blade/src/tokens/theme/theme.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (nashcheez) [reply packages/blade/src/tokens/theme/theme.ts:151]: @coderabbit How stupid are you?

**Pull request merged**